fiftyone/utils/beam.py

```
@@ -338,6 +345,255 @@
             )
 
 
+def beam_map(
```

Pylint (3.3.7)

[refactor] Too many positional arguments (11/5)

(R0917)